content/nginxaas-azure/version_changelog.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Version Changelog"
+weight: 950
+toc: true
+docs: "DOCS-870" #need to update
```

Comment:
Should I create a new page for the version changelog or keep the version information on our regular Changelog page?